src/pages/deploy/exposing-your-app.md

```diff
@@ -2,30 +2,40 @@
 title: Exposing Your App
 ---
 
-Before your application can say hello, Railway needs to know what PORT to listen on to expose your application to the internet. Railway does try to do it's best to do this automatically for you however, there are cases when we can't.
+Before your application can say hello, Railway needs to know the IP and port that your application is listening on, in order to expose it to the internet.
 
-You can configure your application to use the `PORT` environment variable by adding the `PORT` on your service's variables page. (Command + K or CTRL + K, depending on your keyboard/OS and type `Variables` or you can use the keyboard shortcut: `G` + `V` under your selected service)
+The easiest way to get up and running is to have your application listen on `0.0.0.0:$PORT`, where `PORT` is a Railway-provided environment variable. 
 
-**A Note on Listening IPs**: It's best for your application to listen on `0.0.0.0:$PORT`. While most things work with `127.0.0.1` and localhost, some do not (Django for example)
+Alternatively, you can manually override the `PORT` environment variable by adding `PORT` to your projects variables page. (Command + K and type `Variables` or you can use the keyboard shortcut: `G` + `V` under your selected project)
 
-Each Railway environment will automatically be configured with a public Railway
-domain, which looks like `*.up.railway.app`. This can be changed at any time if
-a custom domain has not yet been configured.
+Each Railway environment will automatically be configured with a public Railway domain, which looks like `*.up.railway.app`. This can be changed at any time if a custom domain has not yet been configured.
```

Comment:
This statement feels contradictory to the Railway provided domain. We should say, we assign a domain when we notice the application has a server.

Reply:
Oh ya, I had deleted this paragraph at one point. Not sure how it snuck back in. Good catch!